radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/RadixEngineStateComputer.java

```diff
@@ -398,6 +404,7 @@ public void commit(VerifiedTxnsAndProof txnsAndProof, VerifiedVertexStoreState v
 			outputBuilder.put(EpochChange.class, e);
 		});
 		outputBuilder.put(REOutput.class, REOutput.create(txCommitted));
+		outputBuilder.put(RadixEngineResult.class, radixEngineResult);
```

Comment:
Seems wrong to include the entire object here, better would be to set `ForkConfig` received from `.commitInternal()`

Reply:
I don't quite like to include it without any context. On the reader side it's hard to reason about it if we just have a ForkConfig. Is it the current fork config? The next one?